src/components/UpdateAppModal/BaseUpdateAppModal.js

```diff
@@ -1,44 +1,33 @@
-import React, {PureComponent} from 'react';
+import React, {useState} from 'react';
 import {propTypes, defaultProps} from './updateAppModalPropTypes';
 import ConfirmModal from '../ConfirmModal';
 import withLocalize from '../withLocalize';
 
-class BaseUpdateAppModal extends PureComponent {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            isModalOpen: true,
-        };
-
-        this.submitAndClose = this.submitAndClose.bind(this);
-    }
+function BaseUpdateAppModal(props) {
```

Comment:
Use destructuring to get the props -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring